src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -173,6 +173,12 @@ public virtual string CustomMethod
         #region Proxy
 
         /// <summary>
+        /// gets or sets the NoProxy property
+        /// </summary>
+        [Parameter]
+        public virtual SwitchParameter NoProxy { get; set; }
```

Comment:
To be honest, I copied what was already used in that file for the other switches present.